java/de/yessoft/android/mohammad/ui/MainActivity/fragments/NewsFeedRV/NewsCardVH.java

```diff
@@ -0,0 +1,34 @@
+package de.yessoft.android.mohammad.ui.MainActivity.fragments.NewsFeedRV;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.AppCompatImageView;
+import androidx.recyclerview.widget.RecyclerView;
+
+import de.yessoft.android.mohammad.R;
+
+public class NewsCardVH extends RecyclerView.ViewHolder {
+
+    TextView tvSource; // The White on Colour Text
```

Comment:
variable names as before, we should  be consistent in choosing the names

Reply:
I can see the point, Gonna do that